backend/Controllers/address.js

```diff
@@ -0,0 +1,53 @@
+const _ = require('lodash');   
+exports.getAddress = async (req, res) => {
+    let {address} = req
+    try {
+        res.json({success:true, response: address})
```

Comment:
instead of using an extra `success` parameter, use status code to determine whether the API call failed or not. See http://expressjs.com/en/api.html#res.status